pkg/kyverno/test/test_command.go

```diff
@@ -166,6 +227,10 @@ func Command() *cobra.Command {
 	}
 	cmd.Flags().StringVarP(&fileName, "file-name", "f", "kyverno-test.yaml", "test filename")
 	cmd.Flags().StringVarP(&gitBranch, "git-branch", "b", "", "test github repository branch")
+
+	cmd.Flags().BoolP("manifest-mutate", "m", false, "printing out a template test manifest for mutate  policy")
```

Comment:
Where are these implemented and what's their output?

Reply:
There are two spaces between "mutate" and "policy".

Reply:
> Where are these implemented 

I couldn't get that. Sorry!

Reply:
Disregard previous statement. I don't understand why there are three rather than just two manifest output options.

Reply:
The description in English needs a bit of polish:  "prints out a template test manifest for a mutate policy." Same for the other.

Reply:
I would make changes as per your suggestion. Thanks

Reply:
You're still writing "printing" which is a present participle. It should be "prints" because it's not a continual action. It occurs once if a user includes that flag, not forever more.

Reply:
Yes I understood that, I will be making changes as I told. Let me know if everything else is okay.